### PR TITLE
Awesome books: ES6

### PR DESCRIPTION
**Pull request by alexpuente0**
##On this pull request we:

- refactored the awesome books project to use es6 syntax
- used modules for the JS code
- used Luxon Library
- refactored common functions to arrow functions

### CODE REVIEW

File: modules/books.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  }
+
+  static remove(element) {
+    function removeBook() {
+      booksArray = booksArray.filter((book) => +book.idNum 
+      !== +this.parentNode.children[0].children[2].innerHTML);
+      this.parentNode.remove();
+
+      booksArray.forEach((book, i) => {
+        bookList.children[i].children[0].children[2].innerHTML = i;
```
Comment (brytebee) [inline modules/books.js]: - [x] **Required:** This is working as it should. Well done. However, you are required to favour the use of ES6 arrow functions where possible.

image

File: modules/sections.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export function changeSection() {
```
Comment (brytebee) [inline modules/sections.js]: - [x] **Required:** Kindly change this function as well to use `ES6 arrow function`.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @alexpuente0👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Project correctly set up.✔️
- There is a `modules` directory with other .js files.✔️
- The app is working correctly.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alexpuente0** [comment]
hello, @brytebee !

thank you for the kind feedback!
I actually tried to change those functions into arrow functions before, but when I do, the THIS becomes unusable (undefined), which messes up my code, do you have an idea on how to fix it?

File: modules/books.js (hunk 1 above)
Comment (alexpuente0) [reply modules/books.js]: It was really complicated and I had to change quite a few things but I made it work and learned a lot, thanks!

File: modules/sections.js (hunk 2 above)
Comment (alexpuente0) [reply modules/sections.js]: Fixed this as well, thank you!

**Review by elyor-doniyorov — APPROVED** [review]
Hi @alexpuente0, 

In my opinion, your project milestone is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by alexpuente0** [comment]
thank you @elyor-doniyorov !

**Pull request merged**